spec/cli_v1_commands.yaml

```diff
@@ -45,6 +45,256 @@ flags:
+            --project-ref abcdefghijklmnopqrst \
+            --add-domains company.net
+        response: |-
+          Information about the updated provider.
+      - id: remove-domains
+        name: Remove a domain
+        code: |-
+          supabase sso update 6df4d73f-bf21-405f-a084-b11adf19fea5 \
+            --project-ref abcdefghijklmnopqrst \
+            --remove-domains company.org
+        response: |-
+          Information about the updated provider.
+    flags:
+      - id: metadata-url
+        name: --metadata-url <URL>
+        description: |-
+          A URL (typically HTTPS) that points to a SAML 2.0 Metadata XML document describing the SAML features supported by the identity provider being added.
+          Mutually exclusive with --metadata-file.
+        type: string
+      - id: metadata-file
+        name: --metadata-file <path to file>
+        description: |
+          A path to a file containing a SAML 2.0 Metadata XML document describing the SAML features supported by the identity provider being added. Prefer using --metadata-url if possible.
+
+          Mutually exclusive with --metadata-file.
+        type: string
+      - id: domains
+        name: domains
+        description: |-
+          A comma separated list of email domains to associate with this identity provider. Used to quickly sign users in on the frontend by identifying the correct identity provider based on the email address of the user. When provided all of the domains already attached to the identity provider will be replaced with this list.
+        type: string
+      - id: add-domains
+        name: add-domains
+        description: |-
+          Associates these additional email domains to the existing set of domains. See --domains for a detailed explanation.
+      - id: remove-domains
+        name: remove-domains
+        description: |-
+          Removes these email domains from the existing set of domains. See --domains for a detailed explanation.
+      - id: attribute-mapping-file
+        name: attribute-mapping-file
+        type: string
+        description: |-
+          A file containing a JSON document describing the attribute mappings attached to this identity provider. User information contained in the SAML assertion will be mapped according to the rules laid out in this document.
+  - id: supabase-sso-list
+    title: supabase sso list
+    summary: Add a connection to a SSO identity provider
+    description: |-
+      Add and configure a new connection to a SSO identity provider to your Supabase project.
+    tags:
+      - management-api
+    links: []
+    flags: []
+  - id: supabase-sso-show
+    title: supabase sso show <provider-id>
+    summary: Show information about an identity provider.
+    description: |-
+      Provides the information about an established connection to an identity provider. You can use --output metadata to obtain the raw SAML 2.0 Metadata XML document stored in your project's configuration.
```

Comment:
Do double check this, can't seem to find the `--output metadata` option on v1.49.4 (latest version on brew)

<img width="986" alt="CleanShot 2023-04-06 at 20 47 33@2x" src="https://user-images.githubusercontent.com/8011761/230383070-d58ffe6c-5bf2-4967-9d17-3a10a615d5f9.png">

Reply:
Ah yes got a bit confused. Will fix the whole thing.